kafka_consumer/README.md

```diff
@@ -1,32 +1,54 @@
-# Kafka_consumer Integration
+# kafka_consumer Integration
 
-## Overview
+# Overview
 
-Get metrics from kafka_consumer service in real time to:
+This Agent check only collects metrics for Kafka broker offset, consumer offset, and consumer lag. If you want to collect many more Kafka metrics, see the kafka check.
```

Comment:
This is a bit unclear, simply because of the kafka terminology.

Maybe:
> This Agent check only collects metrics for message offsets. If you want to collect
metrics about the Kafka brokers themselves, see the `kafka`[hyperlink to it] check.
Currently this check fetches the highwater offsets from the Kafka brokers, consumer offsets for old-style consumers that store their offsets in zookeeper, and the calculated consumer lag which is the difference between those two metrics. 
>
> Note: This check does not currently support collecting consumer offsets for new-style consumer groups which store their offsets in Kafka. Supporting for that is tracked in https://github.com/DataDog/integrations-core/issues/457.